src/windows/main_window.py

```diff
@@ -1021,6 +1021,12 @@ def actionSaveFrame_trigger(self, event):
             self.statusBar.showMessage(_("Save Frame cancelled..."), 5000)
             return
 
+        # Set scale mode to higher quality (to skip chroma optimizations of preview)
+        openshot.Settings.Instance().HIGH_QUALITY_SCALING = True
+
+        # Clear current frame from cache to get export-like quality image
+        self.timeline_sync.timeline.Remove(self.preview_thread.current_frame)
```

Comment:
```
Traceback (most recent call last):
  File "/home/ferd/rpmbuild/REPOS/openshot-qt/worktree-testing/src/windows/main_window.py", line 1028, in actionSaveFrame_trigger
    self.timeline_sync.timeline.Remove(self.preview_thread.current_frame)
AttributeError: 'Timeline' object has no attribute 'Remove'
```

...I don't think it should be necessary to flush the cache anyway. The entire cache gets _replaced_ with a temporary one a few lines later, **before** the call to save the image to disk. So, whatever's in the cache when this line is executed isn't what gets saved anyway.

Reply:
https://github.com/OpenShot/openshot-qt/blob/accfd6a137bcc6bbfc8986a383f0705f8ae6ff75/src/windows/main_window.py#L1034-L1041

Reply:
I copied wrong code here. But...

> The entire cache gets _replaced_ with a temporary one a few lines later, **before** the call to save the image to disk...

this doesn't work as it written (or intended to work).

Reply:
More over, the https://github.com/OpenShot/libopenshot/commit/968e472c73bccac9179a139fa7584a05acf8a95b
set some limits on: `Timeline::SetCache` method, so mentioned "replace" is questionable thing. VERY questionable THING.

I think that I need simple:
```
        # Clear timeline preview cache (to set new quality)
        self.timeline_sync.timeline.ClearAllCache()
```
 (as I actually did in my own code) This thing is surely works.